### DISCUSSION TITLE
MageRun as per PHP/Magento version

### DISCUSSION THREAD

**Discussion by MagePsycho** [wardenenv/warden#743] [Ideas]
I see that MageRun version is fixed in the container.
Can we make MageRun as per the PHP or Magento version so that it runs even for older versions of PHP/Magento?

Or is there a starting point where we can help contributing to it?

**Comment by navarr** [comment]
I didn't realize Magerun was dropping support for Magento versions.  Yes, I think that's a good idea.

It looks like, due to some unique version combinations, we'd have to set it up similarly to how we do Node.JS - which isn't going to be easy or fun.
